### ISSUE TITLE
Watched videos not hidden

### ISSUE THREAD

**Issue by netjeff** [EvHaus/youtube-hide-watched#15]
First time user.  Installed, reloaded, but watched videos are still showing in my https://www.youtube.com/feed/subscriptions.   

Here's my env
* Firefox 57.0.2, 64-bit, on Windows 10
* version 2.5 of this script
* Tampermonkey 4.5.5648
* all other addons disabled

FYI, I am _not_ using the old-style youtube. 

I edited the script to enable debugging
```var __DEV__ = true;```

Here is the output in the browser console when I reload the page
```
[YT-HWV] Starting Script                   subscriptions%20line%202%20%3E%20Function:278
[YT-HWV] Attaching DOM listener            subscriptions%20line%202%20%3E%20Function:256
[YT-HWV] Running check for watched videos  subscriptions%20line%202%20%3E%20Function:225
[YT-HWV] Running check for watched videos  subscriptions%20line%202%20%3E%20Function:225
```

**Comment by EvHaus** [comment]
Did you press this button?

![button](https://user-images.githubusercontent.com/226640/33799200-dee51ef8-dcdb-11e7-9489-a5e9b87dec17.png)

**Comment by netjeff** [comment]
First, I did not realize that the script would add a button to the standard youtube buttons.   Might be worth mentioning in the [README.md](https://github.com/EvHaus/youtube-hide-watched/blob/master/README.md)

=============

Second, the button is invisible.  If I hover over the place where the button should be, a tooltip appears, and if I click, the invisible button the watched videos toggle hidden: 

![untitled](https://user-images.githubusercontent.com/3266932/33800440-e78b87f6-dcfc-11e7-860d-aede68ad9c08.jpg)

=============

If I stack another window over the Firefox window, then the button appears:

![image](https://user-images.githubusercontent.com/3266932/33800475-654ab806-dcfd-11e7-9c43-95b228ddb72d.png)

**Comment by EvHaus** [comment]
Please try again in version 2.6: https://greasyfork.org/en/scripts/13040-youtube-hide-watched-videos

**Comment by netjeff** [comment]
Confirmed fixed for me in v2.6.  Thanks!
